packages/orchestration/src/chain-info.js

```diff
@@ -66,7 +67,21 @@ const knownChains = /** @satisfies {Record<string, ChainInfo>} */ (
 /** @typedef {typeof knownChains} KnownChains */
 
 /**
- * @param {ERef<import('@agoric/vats').NameHubKit['nameAdmin']>} agoricNamesAdmin
+ * TODO: include this in registerChain
+ *
+ * @param {ERef<NameAdmin>} agoricNamesAdmin
+ * @param {string} name
+ * @param {CosmosAssetInfo[]} assets
+ */
+export const registerChainAssets = async (agoricNamesAdmin, name, assets) => {
+  mustMatch(assets, M.arrayOf(CosmosAssetInfoShape));
+  const { nameAdmin: assetAdmin } =
+    await E(agoricNamesAdmin).provideChild('chainAssets');
```

Comment:
please source from a constant,
https://github.com/Agoric/agoric-sdk/blob/3845eacd2691a7dfadd0f96aa723827ae07153e9/packages/orchestration/src/exos/chain-hub.js#L25-L29

The other `provideChild` calls in this function should have been as well. Please improve while here